app/assets/stylesheets/navbar.css

```diff
@@ -169,6 +169,7 @@ img.avatar {
 
 .container, .container-fluid{
     max-width: 1200px;
+    width: 100% !important;
```

Comment:
Why is this change assigned to this ticket? - I think it belongs to #183.

Reply:
That was part of Dimas requested changes for the event tiles as bootstrap somehow always set the container width to 70%.

Reply:
FYI: The container stuff is generally wanted and comes from Materialize.